backend/core/src/migration/1692890494691-ami-120-updates.ts

```diff
@@ -94,12 +94,15 @@ export class ami120Updates1692890494691 implements MigrationInterface {
         FROM ami_chart
         WHERE name='${amiUpdate.name}'`
       )
-      const combinedItems = [...currentItems, ...amiUpdate.items]
-      await queryRunner.query(
-        `UPDATE ami_chart 
+      if (currentItems[0]) {
```

Comment:
is there a case where this is false?

Reply:
yeah, if the ami chart doesn't exist in the system this will be false. It's testable if you run this migration without this line against the seeded data since none of the above ami-charts exist